enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -457,7 +470,8 @@ impl<'a> Handler<'a> {
         let trusted = unsafe { self.aex.gpr.rsi.into_slice_mut(1usize) };
 
         let cursor = self.block.cursor();
-        let untrusted = unsafe { cursor.alloc::<libc::timespec>(1).or(Err(libc::EMSGSIZE))? };
+        let (_cursor, untrusted) =
```

Comment:
```suggestion
        let (_, untrusted) =
```

Discard with naked underscore.